index.js

```diff
@@ -126,10 +128,7 @@ var new_client = function(api_key, config) {
       agent: this.proxy_agent
     };
 
-    var make_request = (function(cb, err_cb) {
-      requestify.request(this.base_uri + '/api/eval/features/' + key, request_params)
-      .then(cb, err_cb);
-    }).bind(this);
+    var request = make_request(client, '/api/eval/features/' + key);
```

Comment:
Identifier 'make_request' is not in camel case.<br>Too many errors. (25% scanned).